apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabUpgradeOpt/upOpt.ts

```diff
@@ -186,14 +186,14 @@ export class UpOptCalculator {
     const evalOpt = optimize(toEval, {}, ({ path: [p] }) => p !== 'dyn')
 
     const evalFn = precompute(evalOpt, {}, (f) => f.path[1], 5)
-    thresholds[0] = evalFn(Object.values(build))[0] // dmg threshold is current objective value
+    thresholds[0] = evalFn(Object.values(build) as any)[0] // dmg threshold is current objective value
```

Comment:
Not sure why there are some issues here, since nothing about upOpt has been changed. probably need to get this resolved before we merge this in